amethyst_renderer/src/formats/mod.rs

```diff
@@ -9,34 +9,63 @@ use amethyst_assets::{AssetPrefab, Format, PrefabData, PrefabError, ProgressCoun
 use amethyst_core::specs::prelude::Entity;
 use serde::{Deserialize, Serialize};
 
-use {Mesh, Texture};
+use shape::InternalShape;
+use {Mesh, ShapePrefab, Texture};
 
 mod mesh;
 mod mtl;
 mod texture;
 
+/// Internal mesh loading
+///
+/// ### Type parameters:
+///
+/// `V`: Vertex format to use for generated `Mesh`es, must to be one of:
```

Comment:
"must to be one of" doesn't quite sound right, should probably read "must be one of"